content/en/blog/2025/sig-etcd-spotlight/sig-etcd-spotlight.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: blog
+title: "Spotlight on SIG etcd"
+slug: sig-etcd-spotlight
+date: 2025-02-05
+author: "Frederico Muñoz (SAS Institute)"
+---
+
+In this SIG etcd spotlight we talked with [James Blair](https://github.com/jmhbnz), [Marek
+Siarkowicz](https://github.com/serathius), [Wenjia Zhang](https://github.com/wenjiaswe), and
+[Benjamin Wang](https://github.com/ahrtr) to learn a bit more about this Kubernetes Special Interest
+Group.
+
+## Introducing SIG etcd
+
+**Frederico: Hello, thank you for the time! Let’s start with some introductions, could you tell us a
+bit about yourself, your role and how you got involved in Kubernetes.**
```

Comment:
```suggestion
**Frederico:** Hello, thank you for the time! Let’s start with some introductions, could you tell us a
bit about yourself, your role and how you got involved in Kubernetes.
```

Reply:
This is on purpose: there was some difficulty in differentiating the questions from the answers (especially when multiple people are also answering), so like previous spotlights I've opted to put the entire question on bold. I'll resolve this one but we can reopen if  change is needed.